test/functional/testlib.bash

```diff
@@ -1355,7 +1390,9 @@ create_version() { # swupd_function
 	sudo mkdir -p "$env_name"/web-dir/"$version"/{files,delta}
 	sudo mkdir -p "$env_name"/web-dir/version/format"$format"
 	write_to_protected_file "$env_name"/web-dir/version/format"$format"/latest "$version"
+	sign_version "$env_name"/web-dir/version/format"$format"/latest format
```

Comment:
are you just passing the string "format" to the function?

Reply:
Actually I don't really need to pass a format to this function. Just need to pass something. If the second argument is empty, it means it is signing latest_version.

Now that you pointed it makes sense to regex a pattern in the function without the need for a second argument.